apps/backoffice-v2/src/pages/Entities/components/CaseCreation/components/CaseCreationForm/hooks/useCaseCreationForm/utils/create-context-from-form-data.ts

```diff
@@ -0,0 +1,16 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { generateEntityId } from '@/pages/Entities/components/CaseCreation/components/CaseCreationForm/hooks/useCaseCreationForm/utils/generate-entity-id';
+import { AnyObject } from '@ballerine/ui';
+
+export const createContextFromFormData = (formData: AnyObject): TWorkflowById['context'] => {
+  const context = {
+    entity: {
+      id: generateEntityId(),
```

Comment:
In case you generate this id on client side we should take care for unique on our backend